src/components/mobile/mobile.vue

```diff
@@ -0,0 +1,143 @@
+<template>
+  <SingleColumnLayout>
+    <template #header>
+      <div
+        class="
+          x-list x-list--horizontal x-list__item--expand x-list--padding-03 x-list--align-center
+        "
+      >
+        <Close class="x-button--ghost">
+          <ArrowIcon />
+        </Close>
+        <SearchBox class="x-list__item--expand" />
```

Comment:
Current searchbox  font size (`14px`) is too small for apple devices, this will trigger a zoom when focused. It needs a minimum of `16px`.

Reply:
yes, we will include multiple buttons, inputs, etc. sizes. But for another task/PR